src/Http/Controllers/VoyagerBaseController.php

```diff
@@ -686,7 +686,7 @@ public function remove_media(Request $request)
                     'status'  => $code,
                     'message' => $message,
                 ],
-            ], $code);
+            ], 500);
```

Comment:
Set HTTP response code to 500 when there is an error instead of the exception code